server/app/forms/ProgramForm.java

```diff
@@ -115,12 +116,21 @@ public void setLocalizedShortDescription(String localizedShortDescription) {
     this.localizedShortDescription = localizedShortDescription;
   }
 
-  public Boolean getIsCommonIntakeForm() {
-    return isCommonIntakeForm;
+  public String getProgramTypeValue() {
+    return programTypeValue;
   }
 
-  public void setIsCommonIntakeForm(Boolean isCommonIntakeForm) {
-    this.isCommonIntakeForm = isCommonIntakeForm;
+  public void setProgramTypeValue(String programTypeValue) {
+    this.programTypeValue = programTypeValue;
+  }
+
+  public ProgramType getProgramType() {
+    for (ProgramType programType : ProgramType.values()) {
+      if (programType.getValue().equals(programTypeValue)) {
+        return programType;
+      }
+    }
+    throw new IllegalArgumentException("Unknown ProgramType for: " + programTypeValue);
```

Comment:
(blocking) This should be moved to be a static method on the ProgramType enum

Reply:
Changed to `ProgramForm` per [comment](https://github.com/civiform/civiform/pull/10100#discussion_r2023216787). 
I do agree that this method better belongs in `ProgramType` enum, however it makes the calls in `AdminProgramController.java` much nicer by moving it to `ProgamForm` (as seen on the last commit). I don't have strong opinions, so happy to change it back